### ISSUE TITLE
Select2 Control appearance is buggy

### ISSUE THREAD

**Issue by latheeshvmv** [elementor/elementor#10850]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

There is already a case: https://github.com/elementor/elementor/issues/10624 but it has not been marked as a bug yet. The select2 control looks like this in elementor latest version. I want to bring attention to this .Maybe #10624  didn't shared much details so it's being ignored. 

![image](https://user-images.githubusercontent.com/44595562/76693083-a00b4c80-6685-11ea-897f-d43fd3435b55.png)

## Steps to reproduce

```
Just use a select control as in this documentation (https://developers.elementor.com/elementor-controls/select2-control/) without a default value
$this->add_control(
			'show_elements',
			[
				'label' => __( 'Show Elements', 'plugin-domain' ),
				'type' => \Elementor\Controls_Manager::SELECT2,
				'multiple' => true,
				'options' => [
					'title'  => __( 'Title', 'plugin-domain' ),
					'description' => __( 'Description', 'plugin-domain' ),
					'button' => __( 'Button', 'plugin-domain' ),
				],
			]
		);
```

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>
					
== Server Environment ==
	Operating System: WINNT
	Software: nginx/1.16.0
	MySQL version: MySQL Community Server - GPL v8.0.16
	PHP Version: 7.4.1
	PHP Max Input Vars: 4000
	PHP Max Post Size: 1000M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.3.2
	Site URL: https://energy.local
	Home URL: https://energy.local
	WP Multisite: No
	Max Upload Size: 300 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.2.2
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.132 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 2.9.6
		Author: Elementor.com

== Elements Usage ==
	
	wp-page : 1

== Log ==

**Comment by gonzalesc** [comment]
Hi.
Does this issue was resolved? I have the same problem.

**Comment by latheeshvmv** [comment]
@gonzalesc  They seems to have gone to lazy mode now, it's been more than a month they didn't bother even to mark it is a bug.
temporary or permanent fix is use this for each select2 field types :
`'label_block'=>true,`

**Comment by alessandrotesoro** [comment]
I just noticed this issue too :(

**Comment by notrealdev** [comment]
I have the same problem. Please fixed it!

**Comment by datfit09** [comment]
Does this issue was resolved? I have the same problem.

**Comment by DorShahar** [comment]
Hi @latheeshvmv  - Thank you for reporting.

I merged this issue with  #10624, which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.
Thanks for your feedback!
